pkg/subctl/operator/deploy/ensure.go

```diff
@@ -52,12 +53,22 @@ func Ensure(config *rest.Config, submarinerNamespace string, repository string,
 		brokerURL = brokerURL[(idx + 3):]
 	}
 
+	if len(repository) == 0 {
+		// Default repository; this is handled in the operator after 0.0.1
+		repository = "quay.io/submariner"
+	}
+
+	if len(version) == 0 {
+		// Default version; this is handled in the operator after 0.0.1
+		version = "0.0.1"
```

Comment:
we need to use 0.0.2,

0.0.1 is the initial rancher release for submariner, right?